msvc/libmonoutils.vcxproj

```diff
@@ -110,6 +110,9 @@
     <ClInclude Include="..\mono\utils\mono-dl.h" />
     <ClInclude Include="..\mono\utils\mono-error-internals.h" />
     <ClInclude Include="..\mono\utils\mono-error.h" />
+    <ClInclude Include="..\mono\utils\mono-log-windows.c" />
+    <ClInclude Include="..\mono\utils\mono-log-common.c" />
+    <ClInclude Include="..\mono\utils\mono-log-posix.c" />

```

Comment:
These should be `<ClCompile Include="..." />` and up in the previous `<ItemGroup>`, not here.

(Windows build is broken).